docs/bucket-notifications/README.md

```diff
@@ -0,0 +1,435 @@
+[![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+Minio server supports Amazon S3 compatible bucket event notification for following targets [AMQP](https://www.amqp.org/about/what), [Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/getting-started.html) , [Redis](http://redis.io/documentation), [nats.io](http://nats.io/) and [PostgreSQL](https://www.postgresql.org).
```

Comment:
inio server supports Amazon S3 compatible bucket event notification for *the* following targets